src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -17,14 +17,19 @@ import { DatePicker } from '@mui/x-date-pickers/DatePicker';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import useMediaQuery from '@mui/material/useMediaQuery';
 import { useTheme } from '@mui/material/styles';
-// Context Imports
+/* import Slide from '@mui/material/Slide';
+ */ // Context Imports
 import { DocumentListContext } from '@contexts';
 // Component Imports
 import { DocumentSelection, FormSection } from '../Form';
 import UploadButtonGroup from './UploadButtonGroup';
 import useNotification from '../../hooks/useNotification';
 import UploadDocumentConfirmationModal from './UploadDocumentConfirmationModal';
-
+/* 
```

Comment:
Is this `Transition` component gonna be used for an up coming PR for modal/drawer animation?